engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/ordering/SortArrayNode.java

```diff
@@ -0,0 +1,32 @@
+package org.enso.interpreter.node.expression.builtin.ordering;
+
+import com.oracle.truffle.api.nodes.Node;
+import org.enso.interpreter.dsl.AcceptsError;
+import org.enso.interpreter.dsl.BuiltinMethod;
+import org.enso.interpreter.runtime.state.State;
+
+/**
+ * Just a wrapper for {@code Array.sort_builtin} that delegates to {@code Vector.sort_builtin}. This
+ * is a temporary workaround until Array and Vector are fully merged into a single type.
```

Comment:
```suggestion
 * Just a wrapper for {@code Array.sort_builtin} that delegates to {@code Vector.sort_builtin}.
```

Is the merger planned? From what I remember from [our discussions with @JaroslavTulach](https://discord.com/channels/@me/948519891961544714/1095320137873227826) we decided that they actually do have slightly different meanings, so we probably want to keep them distinct.

Reply:
Btw. do we need this? Cannot we just make `Array.sort` delegate to `Vector` somehow and have just one builtin method to do this??

Reply:
We cannot, because builtin methods cannot be called as static methods - #6282. I have tried that already.